autopush/web/base.py

```diff
@@ -179,7 +179,9 @@ def _validation_err(self, fail):
         exc = fail.value
         self.log.info(format="Request validation error",
                       status_code=exc.status_code,
-                      errno=exc.errno)
+                      errno=exc.errno,
+                      message=exc.message or "No message",
```

Comment:
we probably want format="Request validation error: %s" % exc). we don't use a message kwarg for any other log calls